### PR TITLE
[Notifier] Add Chatwork Notifier Bridge

### PR DESCRIPTION
**Pull request by Ippey**
| Q             | A
| ------------- | ---
| Branch?       | 6.2
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| License       | MIT

Add Chatwork Notifier Bridge.

[Chatwork](https://go.chatwork.com/) is one of the popular chat tools in Japan.
This bridge allows sending messages to Chatwork via [Chatwork API](https://developer.chatwork.com/docs).

### CODE REVIEW

File: src/Symfony/Component/Notifier/Bridge/Chatwork/ChatworkTransport.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            'headers' => [
+                'X-ChatWorkToken' => $this->apiToken,
+                'Content-Type' => 'application/x-www-form-urlencoded',
+            ],
+        ]);
+
+        try {
+            $statusCode = $response->getStatusCode();
+        } catch (TransportExceptionInterface $e) {
+            throw new TransportException('Could not reach the remote Discord server.', $response, 0, $e);
```
Comment (fabpot) [inline src/Symfony/Component/Notifier/Bridge/Chatwork/ChatworkTransport.php]:
```suggestion
            throw new TransportException('Could not reach the remote Chatwork server.', $response, 0, $e);
```

File: src/Symfony/Component/Notifier/Bridge/Chatwork/ChatworkTransport.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        }
+
+        if (200 !== $statusCode) {
+            $originalContent = $message->getSubject();
+            $result = $response->toArray(false);
+            $errors = $result['errors'];
+            throw new TransportException(sprintf('Unable to post the Chatwork message: "%s" (%d: "%s").', $originalContent, $statusCode, implode(', ', $errors)), $response);
```
Comment (fabpot) [inline src/Symfony/Component/Notifier/Bridge/Chatwork/ChatworkTransport.php]:
```suggestion
            throw new TransportException(sprintf('Unable to post the Chatwork message: "%s" (%d: %s).', $originalContent, $statusCode, implode(', ', $errors)), $response);
```

If you received an error from fabbot here, that's a false positive.

File: src/Symfony/Component/Notifier/Bridge/Chatwork/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+Chatwork Notifier
+===============
```
Comment (fabpot) [inline src/Symfony/Component/Notifier/Bridge/Chatwork/README.md]:
```suggestion
=================
```

**Review by fabpot — CHANGES_REQUESTED** [review]

**Comment by Ippey** [comment]
@fabpot Thank you, I applied those suggestions.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
@Ippey Can you open a PR on symfony/recipes (like this [one](https://github.com/symfony/recipes/pull/1108/files) for instance).

**Comment by fabpot** [comment]
Thank you @Ippey.

**Comment by Ippey** [comment]
@fabpot Created PR.

https://github.com/symfony/recipes/pull/1121

**Pull request merged**